pkg/cmd/kind/load/docker-image/docker-image.go

```diff
@@ -169,9 +177,10 @@ func loadImage(imageTarName string, node nodes.Node) error {
 	return nodeutils.LoadImageArchive(node, f)
 }
 
-// save saves image to dest, as in `docker save`
-func save(image, dest string) error {
-	return exec.Command("docker", "save", "-o", dest, image).Run()
+// save saves images to dest, as in `docker save`
+func save(images []string, dest string) error {
+	commandArgs := strings.Split(fmt.Sprintf("save -o %s %s", dest, strings.Join(images, " ")), " ")
```

Comment:
nit: this seems a little convoluted

```suggestion
	commandArgs := []string{"save", "-o", dest, strings.Join(images, " ")}
```